# Rapid spread of a reassortant A/H3N2 virus during the 2017-2018 influenza season

#### Barney Potter<sup>1</sup>, Rebecca Garten<sup>2</sup>, James Hadfield<sup>1</sup>, John Huddleston<sup>1,3</sup>, Richard A. Neher.<sup>4,5</sup>, Trevor Bedford<sup>1</sup>, and David Wentworth<sup>2</sup>

<sup>1</sup>Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center, Seattle, WA, USA, <sup>2</sup>Influenza Division, National Center for Immunization and Respiratory Diseases (NCIRD), Centers for Disease Control and Prevention (CDC), Atlanta, GA, USA, <sup>3</sup>Molecular and Cellular Biology program, University of Washington, Seattle, WA, USA, <sup>4</sup>Biozentrum, University of Basel, Switzerland, <sup>5</sup>SIB Swiss Institute of Bioinformatics, Basel, Switzerland

## Abstract
The 2017-2018 influenza season caused more hospitalizations and deaths than any year since the 2009 H1N1 pandemic.
The majority of recorded influenza infections were caused by A/H3N2 viruses, with most of the virus's North American diversity falling into the A2 clade.
Within A2, we observe a subclade which we call A2/re that rose to comprise almost 70\% of circulating A/H3N2 viruses by early 2018.
Unlike most fast-growing clades, however, A2/re contains no amino acid substitutions in the HA segment, which normally precipitate rapid clade growth.
Moreover, the emergent clade does not demonstrate any drastic antigenic differences---as measured by haemagglutinin inhibition assay---normally indicative of selective origin.
Rather, we observe that the A2/re clade was the result of a reassortment that occurred in late 2016 or early 2017 and involved the combination of the HA and PB1 segments of an A2-like virus with the other six segments a virus from the clade A1b.
The discovery of this clade shows the need for antigenic analysis that targets neuraminidase in addition to haemagglutinin.
Furthermore, our results illustrate the need for more thorough tracking of full genomes, in order to better understand the seasonal turnover of influenza viruses and to understand the dynamics of influenza epidemics.
